ProcessLib/LIE/HydroMechanics/HydroMechanicsProcess.cpp

```diff
@@ -489,9 +490,22 @@ void HydroMechanicsProcess<GlobalDim>::computeSecondaryVariableConcrete(
 
     // compute nodal w and aperture
     auto const& R = _process_data.fracture_property->R;
+    auto const& b0 = _process_data.fracture_property->aperture0;
     MeshLib::PropertyVector<double>& vec_w = *_process_data.mesh_prop_nodal_w;
     MeshLib::PropertyVector<double>& vec_b = *_process_data.mesh_prop_nodal_b;
 
+    auto compute_nodal_aperture = [&](std::size_t const node_id,
+                                      double const w_n) {
+        // skip aperture computation for element-wise defined b0 because there
+        // are jumps on the nodes between the element's values.
+        if (dynamic_cast<MeshElementParameter<double> const*>(b0))
+            return std::numeric_limits<double>::quiet_NaN();
+
+        ProcessLib::SpatialPosition x;
+        x.setNodeID(node_id);
+        return w_n + (*b0)(0, x)[0];
```

Comment:
What about time, just for completeness? `(0, x)` --> `(t, x)`

Reply:
Here, it's other way around; b0 is b at time 0. Just for safety...
Would be nice if we could force the parameter to be time-independent....

Reply:
Maybe add a comment that time is zero deliberately?
Could be checked during initialization...